src/plugins/intel_cpu/src/emitters/plugin/x64/jit_dnnl_emitters.cpp

```diff
@@ -45,14 +45,12 @@ jit_dnnl_emitter::jit_dnnl_emitter(jit_generator* host,
 
 void jit_dnnl_emitter::set_injector() {
     if (host_isa_ == cpu::x64::sse41) {
-        eltwise_injector_sse42 =
-            std::make_shared<jit_uni_eltwise_injector_f32<cpu::x64::sse41>>(h, kind, alpha, beta, 1.f);
+        eltwise_injector_sse42 = std::make_shared<jit_uni_eltwise_injector<cpu::x64::sse41>>(h, kind, alpha, beta, 1.f);
```

Comment:
what about "std::make_shared<jit_uni_eltwise_injector<cpu::x64::sse41>>(h, kind, alpha, beta, 1.f,data_type::f32 )" to explicitly specify data type even the default parameter is fp32. and apply to all the other cases in this pr.

Reply:
> what about "std::make_shared<jit_uni_eltwise_injectorcpu::x64::sse41>(h, kind, alpha, beta, 1.f,data_type::f32 )" to explicitly specify data type even the default parameter is fp32. and apply to all the other cases in this pr.

specify `data_type::f32`